src/client/auth/auth.go

```diff
@@ -95,6 +100,40 @@ func IsErrNotActivated(err error) bool {
 	return strings.Contains(err.Error(), ErrNotActivated.Error())
 }
 
+// IsErrPartiallyActivated checks if an error is a ErrPartiallyActivated
+func IsErrPartiallyActivated(err error) bool {
+	if err == nil {
+		return false
+	}
+	// TODO(msteffen) This is unstructured because we have no way to propagate
+	// structured errors across GRPC boundaries. Fix
+	return strings.Contains(err.Error(), ErrPartiallyActivated.Error())
+}
+
+// IsErrNotSignedIn returns true if 'err' is a ErrNotSignedIn
+func IsErrNotSignedIn(err error) bool {
```

Comment:
This should probably have a `nil` guard like the other functions do.

Reply:
Done. I added a test for this and the other functions too